internal/storage/block_size.go

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package storage
+
+const KB uint64 = 1024
```

Comment:
We can move this code here too:
```
const (
	B  uint64 = 1
	KB uint64 = 1 << (10 * iota)
	MB
)
```
Otherwise we will have 2 constant defintitions for a KB.

Reply:
There was no use for `B` and `MB` so I just set the value for only `KB`.

Reply:
That is a good point, I will approve the PR.